app/gateway/2.7.x/install-and-run/debian.md

```diff
@@ -19,12 +25,15 @@ badge: oss
 > [11 Bullseye]({{ site.links.download }}/gateway-2.x-debian-bullseye/pool/all/k/){:.install-listing-link}
 >  </span>
 
+The {{site.base_gateway}} software is governed by the
+[Kong Software License Agreement](https://konghq.com/kongsoftwarelicense/).
 {{site.ce_product_name}} is licensed under an
 [Apache 2.0 license](https://github.com/Kong/kong/blob/master/LICENSE).
 
 ## Prerequisites
 
-You have a supported system with root or [root-equivalent](/gateway/{{page.kong_version}}/plan-and-deploy/kong-user) access.
+* A supported system with root or [root-equivalent](/gateway/{{page.kong_version}}/plan-and-deploy/kong-user) access.
+* (Enterprise only) A `license.json` file from Kong.
 
```

Comment:
Minor note: in order to run the installation commands below, there's a prereq that these packages are installed:

* `curl`
* `lsb-release`
* `apt-transport-https` (if installing the apt repository)

They are insanely common packages, so most folks should have them installed already (though the base images from docker hub did not), and in my opinion it is reasonable to assume that the average Kong user will be able to figure out very quickly how to install them if they are missing.

I'm probably 50/50 on if these needs to be included as an extra step. I'll leave it up to you on what balance to strike between terse and verbose.

Reply:
If you _do_ decide to add this, it should be inserted as step 1:

### Package

1. Install dependencies

```
sudo apt-get install curl lsb-release
```


### APT Repository

1. Install dependencies

```
sudo apt-get install curl lsb-release apt-transport-https
```

Reply:
Good callout. If anything, we should probably add these to _all_ of our install docs - one of the outstanding tasks we have is to actually write useful prereqs, and you just did some of the work for us 😄

Reply:
I'll include it as prerequisites instead of adding a step. Otherwise we can easily get into muddled territory about what we should be documenting and what we shouldn't.